cg/utils/flow_cell.py

```diff
@@ -1,3 +1,5 @@
 def get_flow_cell_id(flow_cell_dir_name: str) -> str:
```

Comment:
GitHub is weird... I just found that this function was duplicated and used in different parts of the code so I kept only one version and moved it to a new module `cg/utils/flow_cell.py`